ui/src/app/views/cohort-edit/component.ts

```diff
@@ -15,6 +15,7 @@ export class CohortEditComponent implements OnInit {
   adding = false;
   workspaceNamespace: string;
   workspaceId: string;
+  buttonClicked = false;
```

Comment:
In addition / as an alternative to this state, it would be good to change the button so it appears disabled. (Or actually doesn't handle onclick events anymore.)